lib/config_cat/constants.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ConfigCat.Constants do
+
+  @feature_flags "f"
+  @comparator "t"
+  @comparison_attribute "a"
+  @comparison_value "c"
+  @rollout_rules "r"
+  @percentage_rules "p"
+  @percentage "p"
+  @value "v"
+  @variation_id "i"
```

Comment:
#minor We could inline these module attributes into the macros below, since we're only using them once.  e.g. `defmacro feature_flags, do: "f"`

Reply:
Done!